ui.go

```diff
@@ -1332,6 +1355,17 @@ func readCmdEvent(ev tcell.Event) expr {
 			}
 		} else {
 			val := gKeyVal[tev.Key()]
+			if len(val) > 0 && val[0] == '<' {
```

Comment:
I think that this should be a separate function (as long as we need this functionality in two places).

Reply:
Done in a484411, thanks for the suggestion. I cleaned up the code slightly as well.